packages/blocks/src/page-block/edgeless/components/toolbar/edgeless-toolbar.ts

```diff
@@ -68,21 +69,33 @@ export function launchIntoFullscreen(element: Element) {
   }
 }
 
+const hideToolbarKey = 'blocksuite:' + 'presentation' + ':hideToolbar';
```

Comment:
Better be `const HIDE_TOOLBAR_KEY = 'blocksuite:edgeless:hideToolbar`

And note empty line spaces.